docs/UserGuide.md

```diff
@@ -279,284 +286,345 @@ If your changes to the data file makes its format invalid, F.A.K.E.J.A.R.V.I.S.
 * `addcomment n/Wesley a/Rune Reading c/Not bad` adds the comment “Not bad” on Wesley’s Rune Reading assignment.
 
 
-### 👀Viewing a comment: `viewcomment`
-Shows the comment tagged to a student’s assignment.
+### 👀Viewing a Comment: `viewcomment`
+
+You can view the comment(s) tagged to your student’s assignment.
 
 Format: `viewcomment n/STUDENT_NAME a/ASSIGNMENT`
 
-Parameters:
-* STUDENT_NAME The name of the student.
-* ASSIGNMENT The name of the assignment.
+> **Parameters:**
+> * STUDENT_NAME The name of the student.
+> * ASSIGNMENT The name of the assignment.
 
 Examples:
 * `viewcomment n/Rayson a/Functional Expressionism` shows the comment on Rayson’s Functional Expressionism assignment.
 * `viewcomment n/Wesley a/Rune Reading` shows the comment on Wesley’s Rune Reading assignment.
 
 
-### 🛠️Editing a comment: `editcomment`
-Edits the comment on a student’s assignment.
+### 🛠️Editing a Comment: `editcomment`
+
+You can edit the comment of your student’s assignment.
 
 Format: `editcomment n/STUDENT_NAME a/ASSIGNMENT c/COMMENT`
 
-Parameters:
-* STUDENT_NAME The name of the student.
-* ASSIGNMENT The name of the assignment.
-* COMMENT The new comment.
+> **Parameters:**
+> * STUDENT_NAME The name of the student.
+> * ASSIGNMENT The name of the assignment.
+> * COMMENT The new comment.
 
 Examples:
 * `editcomment n/Rayson a/Functional Expressionism c/Decent` changes the comment on Rayson’s Functional Expressionism assignment to “Decent”.
 * `editcomment n/Wesley a/Rune Reading c/Great` changes the comment on Wesley’s Rune Reading assignment to “Great”.
 
 
-### ❌Deleting a comment: `deletecomment`
-Deletes the comment tagged to a student’s assignment.
+### ❌Deleting a Comment: `deletecomment`
+
+You can delete the comment tagged to your student’s assignment.
 
 Format: `deletecomment n/STUDENT_NAME a/ASSIGNMENT`
 
-Parameters:
-* STUDENT_NAME The name of the student.
-* ASSIGNMENT The name of the assignment.
+> **Parameters:**
+> * STUDENT_NAME The name of the student.
+> * ASSIGNMENT The name of the assignment.
 
 Examples:
 * `deletecomment n/Rayson a/Functional Expressionism` deletes the comment on Rayson’s Functional Expressionism assignment.
 * `deletecomment n/Wesley a/Rune Reading` deletes the comment on Wesley’s Rune Reading assignment.
 
+## Graded Test Management
+### 🛠️Editing a Graded Test Score: `editgradedtest`
+
+You can edit your student’s graded test scores.
+
+Format: `editgrade INDEX ra1/READING_ASSESSMENT_1 ra2/READING_ASSESSMENT_2 mt/MIDTERMS f/FINALS pe/PRACTICALEXAM `
+
+> **Parameters:**
+> * INDEX The index of the student.
+> * READING_ASSESSMENT_1 The score of the student's Reading Assessment 1.
+> * READING_ASSESSMENT_2 The score of the student's Reading Assessment 2.
+> * MIDTERMS The score of the student's MidTerms.
+> * FINALS The score of the student's Final Assessment.
+> * PRACTICALEXAM The score of the student's Practical Exams.
+
+
+## Consultation Management
 
 ### 📆Creating a Consult: `createconsult`
```

Comment:
Not sure if we should rename "Consult" into "Consultation" just for the purpose of UG?

e.g. "Creating a Consultation", to standardise with the below

Reply:
oh right, will update it